src/plugins/mmapstorage/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+include (LibAddMacros)
+
+if (DEPENDENCY_PHASE)
+	set (plugin mmapstorage)
+	find_package (ZLIB)
+	if (ENABLE_CHECKSUM AND NOT ZLIB_FOUND)
+		remove_plugin (${plugin} "zlib development files not found")
+	endif ()
+
+	# clean up for dependency phase
+	unset (plugin)
+endif (DEPENDENCY_PHASE)
+
+set (MMAPSTORAGE_SOURCES dynarray.h dynarray.c mmapstorage.h mmapstorage.c)
+
+if (ENABLE_CHECKSUM)
+	add_plugin (mmapstorage
```

Comment:
rename to mmapstorage_zlib? (compilation variant)

Reply:
It is called mmapstorage_crc.